src/integration/java/io/pinecone/integration/dataPlane/UpdateFetchAndQueryServerlessTest.java

```diff
@@ -43,66 +46,14 @@ public static void setUp() throws IOException, InterruptedException {
         if (indexName.isEmpty()) indexName = createNewIndex(pinecone, dimension, indexType, true);
         index = pinecone.createIndexConnection(indexName);
         asyncIndex = pinecone.createAsyncIndexConnection(indexName);
-    }
-
-    @AfterAll
-    public static void cleanUp() {
-        index.close();
-        asyncIndex.close();
-    }
 
-    @Test
```

Comment:
This is a redundant test since the optional+required parameters are tested as a part of another test in this file called updateAllParamsFetchAndQuery for both async and blocking stubs.